themes/digital.gov/layouts/events/single.html

```diff
@@ -42,15 +42,19 @@ <h1>{{- .Title | markdownify -}}</h1>
               </span>
             </p>
 
-            <p class="host">Hosted by {{ .Params.host }} and {{ .Params.event_organizer }}</p>
+              {{- if .Params.community_name -}}
+                <p class="host">Hosted by Digital.gov's {{ .Params.community_name }} Community</p>
+                {{- else -}}
```

Comment:
Let's fix this `{{- else -}}` indent too.

Reply:
Done